files/lab/workshop/content/cloning.md

```diff
@@ -581,7 +575,7 @@ NAME         AGE   PHASE     IP               NODENAME                       REA
 fc34-clone   88s   Running   192.168.123.66   ocp4-worker2.aio.example.com   True
 ~~~
 
-This machine will also be visible from the OpenShift Virtualization console, which you can navigate to using the top "**Console**" button, or by using your dedicated tab if you've created one. You can login using the "**root**" user with the password you've set earliyer, by going into the "**Workloads**" --> "**Virtualization**" --> "**fc34-clone**" --> "**Console**", if you want to try:
+This machine will also be visible from the OpenShift Virtualization tab. Switch to the **full OpenShift console** (%cnvlab-console-url%). There, you can login using the "**root**" user with the password you've set earlier, by going into the "**Virtualization**" → "**Virtual Machines**" → "**fc34-clone**" → "**Console**", if you want to try:
```

Comment:
seems to be missing a step there ... they'll have to login to the console with kubeadmin / %cnvlab-kubeadmin-password% and then login with the root user in the VM Console

Reply:
Not sure what you're referring to. I re-wrote the paragraph with the login steps